app/views/layouts/application.html.haml

```diff
@@ -10,6 +10,9 @@
     - if current_user
       = render 'layouts/navbar'
     .container.main-container
+      - if current_user.admin? && current_user.organization.facebook_access_token_expired
```

Comment:
delegate `facebook_access_token_expired` to `organization` - add a method to `current_user`: `is_expired_facebook_auth?`